packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -265,6 +265,7 @@ export class OdspDocumentService implements IDocumentService {
                 return connection;
             } catch (error) {
                 this.cache.sessionJoinCache.remove(this.joinSessionKey);
+                error.socketDocumentId = websocketEndpoint.id;
```

Comment:
I'd add some safety here and at least test that it's an object and not null.

Reply:
Also can you please check that in all core cases we actually use object derived from CustomErrorWithProps class? As otherwise properties will not propagate to telemetry

Reply:
hmm, I thought we always through uniform errors. I  will add check.

Reply:
Yes, we are throwing created with properties.